src/main/java/net/sf/jabref/importer/fileformat/EndnoteImporter.java

```diff
@@ -85,26 +86,26 @@ public boolean isRecognizedFormat(InputStream stream) throws IOException {
      */
     @Override
     public List<BibEntry> importEntries(InputStream stream, OutputPrinter status) throws IOException {
-        ArrayList<BibEntry> bibitems = new ArrayList<>();
+        List<BibEntry> bibitems = new ArrayList<>();
         StringBuilder sb = new StringBuilder();
-        BufferedReader in = new BufferedReader(ImportFormatReader.getReaderDefaultEncoding(stream));
+        try (BufferedReader in = new BufferedReader(ImportFormatReader.getReaderDefaultEncoding(stream))) {
```

Comment:
Does this really work? Because you close the input stream afterwards and I am not sure whether the `stream` is reused for different importers. Can you please investigate upon that?